lineapy/_alembic/versions/38d5f834d3b7_orig.py

```diff
@@ -0,0 +1,316 @@
+"""orig
+
+Revision ID: 38d5f834d3b7
+Revises: 
+Create Date: 2022-06-27 18:10:07.339148
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+from lineapy.utils.migration import maybe_add_column, maybe_create_table
+
+# revision identifiers, used by Alembic.
+revision = "38d5f834d3b7"
```

Comment:
Would you mind adding some instructions about database migration in contributing.md? I assume these functions were auto-generated by some alembic commands.

Reply:
added to contributing.md. yes the scaffolding in this file is auto-generated.